### PR TITLE
feat: add management command to create mysql db

### PR DESCRIPTION
**Pull request by jnoortheen**

### CODE REVIEW

File: zappa_django_utils/management/commands/create_mysql_db.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    help = "Create a database from settings file prior to migrations."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--user',
+            dest='user',
+            help='Username to be used to create database',
+        )
+        parser.add_argument(
+            '--passwd',
```
Comment (Miserlou) [inline zappa_django_utils/management/commands/create_mysql_db.py]: Change to `password`

**Comment by jnoortheen** [comment]
done.

**Comment by FlipperPA** [comment]
Looks good to me, I don't have MySQL to test against, but make sense. Thanks for the contribution!

**Comment by atenni** [comment]
Related to #1

**Pull request merged**